frame/oracle/src/lib.rs

```diff
@@ -441,19 +442,25 @@ pub mod pallet {
 			asset_id: Self::AssetId,
 			amount: Self::Balance,
 		) -> Result<Self::Balance, DispatchError> {
-			// imagine 10^3 == 1_000 costs 4
-			// so 1 costs 0,
-			// and amount of normalized desired is 10
-			// 10 * 1_000 / 4 = 2_500
-			// so we need 2_500 asset amount to pay for 10 normalized
-			let unit = 10 ^ (T::LocalAssets::decimals(asset_id))?;
-			let price_asset_for_unit: u128 = Self::get_price(asset_id, unit.into())?.price.into();
+			// assume the following parameters:
+			//
+			// asset A has 3 decimal places
+			// 1 unit of asset ACOIN costs 4 units of normalized asset (4_000_000_000_000)
```

Comment:
Not sure what's meant by "normalized" here.
```suggestion
			// 1 unit of asset ACOIN costs 4 units of default quote asset (4_000_000_000_000)
```
?

Reply:
"default quote" is a better way to phrase it indeed, thank you :+1: